cmd/kube-controller-manager/app/options/options.go

```diff
@@ -46,6 +46,7 @@ func NewCMServer() *CMServer {
 			Port:                              ports.ControllerManagerPort,
 			Address:                           "0.0.0.0",
 			ConcurrentEndpointSyncs:           5,
+			ConcurrentServiceSyncs:            5,
```

Comment:
This is the concurrent worker number for handling service sync up, before it is fixed to 10. `workerGoroutines = 10`, as not all services events are needed to talk to cloud provider, like service type is nodeport, or a service update request does not need to update LB info. I think it is reasonable to keep 5.